src/data/regex-quiz.ts

```diff
@@ -94,6 +94,16 @@ const regexQuiz = [
     Explanation:
       "A character class is used to match any one of several characters in a particular position.",
     Link: "https://www.freecodecamp.org/news/regular-expressions-for-beginners/"
+  },
+  {
+    Question: "Which of the following regex pattern can be used to match one or more occurrences of a digit in a string?",
+    Answer: "\d+",
+    Distractor1: "\d*",
+    Distractor2: "\d?",
+    Distractor3: "\d{2}",
```

Comment:
you need to remove the `\` because it is cause linting errors 

```suggestion
    Answer: "d+",
    Distractor1: "d*",
    Distractor2: "d?",
    Distractor3: "d{2}",
```